code_examples/sdk_examples/src/workshop/attester/generateDid.ts

```diff
@@ -12,40 +9,33 @@ export async function createFullDid(
   fullDid: Kilt.DidDocument
 }> {
   const api = Kilt.ConfigService.get('api')
-
-  const mnemonic = Kilt.Utils.Crypto.mnemonicGenerate()
-  const {
-    authentication,
-    keyAgreement,
-    assertionMethod,
-    capabilityDelegation
-  } = generateKeypairs(mnemonic)
-  // Get tx that will create the DID on chain and DID-URI that can be used to resolve the DID Document.
-  const fullDidCreationTx = await Kilt.Did.getStoreTx(
-    {
-      authentication: [authentication],
-      keyAgreement: [keyAgreement],
-      assertionMethod: [assertionMethod],
-      capabilityDelegation: [capabilityDelegation]
-    },
-    submitterAccount.address,
-    async ({ data }) => ({
-      signature: authentication.sign(data),
-      keyType: authentication.type
-    })
-  )
-
-  await Kilt.Blockchain.signAndSubmitTx(fullDidCreationTx, submitterAccount)
-
-  const didUri = Kilt.Did.getFullDidUriFromKey(authentication)
-  const encodedFullDid = await api.call.did.query(Kilt.Did.toChain(didUri))
-  const { document } = Kilt.Did.linkedInfoFromChain(encodedFullDid)
-
-  if (!document) {
+  const mnemonic = process.env.ATTESTER_ACCOUNT_MNEMONIC as string
```

Comment:
This doesn't really make sense to me. The `mnemonic` used to be the one from which the DID is derived, but now it's the attester mnemonic - and the link between this mnemonic and the DID has become completely unclear.

Reply:
@rflechtner Probably because I am also quite unclear what should be happening. using the new `createFromAccount`, can the DID account and the attester still have the same relationship as before?